sstables/sstables.hh

```diff
@@ -697,6 +698,10 @@ public:
         return _run_identifier;
     }
 
+    bool no_partition_tombstones() const {
```

Comment:
I think we need a name which better reflects what `false` really means here: not sure as opposed to definitely no.

Reply:
How about `cannot_have_partition_tombstones`?

cannot => no partition tombstones
!cannot => can => may or may not have partition tombstones

Reply:
Yes, that is better.